AdvancedTaskSpecFlow/AdvancedTaskSpecFlow/StepDefinitions/EditShareSkillStepDefinitions.cs

```diff
@@ -0,0 +1,349 @@
+using AdvancedTaskSpecFlow.Models;
+using AdvancedTaskSpecFlow.Pages.ProfilePageTabs;
+using AdvancedTaskSpecFlow.Utilities;
+using AventStack.ExtentReports;
+using AventStack.ExtentReports.Model;
+using Microsoft.VisualStudio.TestPlatform.CommunicationUtilities;
+using NUnit.Framework;
+using System;
+using TechTalk.SpecFlow;
+
+namespace AdvancedTaskSpecFlow.StepDefinitions
+{
+    [Binding]
+    public class EditShareSkillStepDefinitions
+    {
+        private readonly CommonDriver commonDriver;
+        public EditShareSkillStepDefinitions(CommonDriver commonDriver)
+        {
+            this.commonDriver = commonDriver;
+        }
+
+        public void LoginAndGoToManageListingsPage()
+        {
+            //Open chrome browser
+            commonDriver.driver.Manage().Window.Maximize();
+            //Launch Mars portal
```

Comment:
you might need to see how they can share state across steps

Reply:
Hi Aman,

I've moved the login information into the CommonDriver class instead. Because I use CommonDriver in all the step definition files, I thought it would make sense to move it there. I created a method called Login that will do the login steps and in the respective step definition files, I call the Login method and then navigate to the appropriate pages.